contracts/badger-sett/strategies/native/StrategyCurveLiquidityRewardGaugeBase.sol

```diff
@@ -96,7 +114,7 @@ contract StrategyCurveLiquidityRewardGaugeBase is BaseStrategy {
     }
 
     function getProtectedTokens() external override view returns (address[] memory) {
-        address[] memory protectedTokens = new address[](3);
+        address[] memory protectedTokens = new address[](4);
```

Comment:
why not return a struct?

Reply:
This was a function that already existed in the contract, I just added the token into it.  @dapp-whisperer - any reason for not having a struct here?